### DISCUSSION TITLE
Fedora releases

### DISCUSSION THREAD

**Discussion by nforro** [neoave/mrack#297] [General]
Hello, is there a reason why the last release hasn't been packaged for Fedora (there are unmerged [Packit PRs](https://src.fedoraproject.org/rpms/mrack/pull-requests) for 1.21.0) and why the version in Fedora 41 and rawhide is older than in Fedora <= 40?

**Comment by pvoborni** [comment]
The reason is that the PRs were neglected.

**Comment by nforro** [comment]
Thanks for taking care of it!
